packages/syft/src/syft/core/tensor/__init__.py

```diff
@@ -154,6 +154,10 @@ def create_tensor_ast(client: Optional[AbstractNodeClient] = None) -> Globals:
         ("syft.core.tensor.tensor.Tensor.mean", "syft.core.tensor.tensor.Tensor"),
         ("syft.core.tensor.tensor.Tensor.min", "syft.core.tensor.tensor.Tensor"),
         ("syft.core.tensor.tensor.Tensor.ndim", "syft.core.tensor.tensor.Tensor"),
+        (
+            "syft.core.tensor.tensor.Tensor.annotate_with_dp_metadata",
```

Comment:
MAYBE: I think we could remove this from the AST, do we intend to allow the data scientist to add their own metadata?

Reply:
👍🏾

Reply:
Fixed in: a4a9d04331065e6c5373972bad41ca5488bfeaf8